CMakeLists.txt

```diff
@@ -1,6 +1,30 @@
+cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+
+if(POLICY CMP0025)
+  cmake_policy(SET CMP0025 NEW)
+endif()
+
 project(MONGO_CXX_DRIVER LANGUAGES CXX)
 
-cmake_minimum_required(VERSION 3.2 FATAL_ERROR)
+if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+  if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS "4.9")
```

Comment:
Yup.  If you're running gcc 4.8.x you won't be able to build the driver now, there's a bug with `std::max_align_t` that I ran into yesterday: https://gcc.gnu.org/bugzilla/show_bug.cgi?id=56019